app/models/delivery_zone.rb

```diff
@@ -0,0 +1,14 @@
+class DeliveryZone < ActiveRecord::Base
+  validates :zipcode, presence: true, uniqueness: { case_sensitive: false }
```

Comment:
@kenyonj validating zipcodes has more nuance to it than expecting them to be numeric (e.g. zipcodes with `-` and international postal codes that are alphanumeric).

If we assume that all codes will be American (which is a safe assumption at the point), this PR (https://github.com/thoughtbot/freshfoodconnect/pull/15) adds the [`validates_zipcode`](https://github.com/dgilperez/validates_zipcode) gem.